### ISSUE TITLE
why forked ten_year_rails ? what's the difference?

### ISSUE THREAD

**Issue by silva96** [fastruby/next_rails#10]
I see you are 9 ahead, 25 behind the original gem. What is different in this fork? you should put an explanation in the readme. Like what it the motivation behind, besides changing the name.

**Comment by etagwerker** [comment]
@silva96 Good question. A couple of years ago we were trying to make contributions thu `ten_year_rails` but we weren't making any progress with them. It seemed that the company had decided to archive the repository. That is why we decided to start `next_rails`.

At [FastRuby.io | Rails Upgrade Services](https://fastruby.io) we work with very old Rails applications and `ten_years_rails` has a dependency on `actionview` which makes it difficult to use it with really old versions... so we decided to get rid of that. I think that is the main difference when it comes to the codebase: https://github.com/clio/ten_years_rails/compare/master...fastruby:master

There were more differences between the gems in the past. At some point the team at Clio decided to merge the PRs that we had submitted to the project. 

I hope this answers your question. 

I think it is a good idea to add a note to the README. 👍
